### ISSUE TITLE
Document opaque / transparent

### ISSUE THREAD

**Issue by joelburget** [mortberg/cubicaltt#85]
The keywords `opaque`, `transparent`, and `transparent_all` aren't documented anywhere.

**Comment by mortberg** [comment]
Indeed. We should document them and say that they are experimental.
